octoprint_auth_ldap/ldap.py

```diff
@@ -28,9 +28,13 @@ def get_client(self, user=None, password=None):
             client = ldap.initialize(uri)
             if self.settings.get([REQUEST_TLS_CERT]):
                 self.logger.debug("Requesting TLS certificate")
+                ldap.set_option(ldap.OPT_X_TLS_REQUIRE_CERT, ldap.OPT_X_TLS_DEMAND)
```

Comment:
Why this option duplicate the next line `client.set_option(ldap.OPT_X_TLS_REQUIRE_CERT, ldap.OPT_X_TLS_DEMAND)` ?

Reply:
Yeah you are right. Only the `ldap.set_option(ldap.OPT_X_TLS_REQUIRE_CERT, ldap.OPT_X_TLS_DEMAND)` is requiered.
If set only for the client e.g. `client.set_option(ldap.OPT_X_TLS_REQUIRE_CERT, ldap.OPT_X_TLS_NEVER)` I get teh following response from our ldap server: `ldap.SERVER_DOWN: {'result': -1, 'desc': "Can't contact LDAP server", 'ctrls': [], 'info': '(unknown error code)'}`.